addons/block_code/ui/blocks/utilities/background/background.gd

```diff
@@ -21,18 +27,36 @@ var parent_block: Block
 @export var shift_bottom: float = 0.0:
 	set = _set_shift_bottom
 
+## Style of the top knob
+@export var top_variant: int = 0:
+	set = _set_top_variant
+
+## |0|, \1/, /2/, <3>, >4>, v5v, v6^, \7y, /8y
```

Comment:
We really don't need the 9 shapes that the first commit adds, and they make the code very hard to read. We only need:
- A pointy background for value blocks of variant type BOOLEAN.
- A round ellipse for entry blocks.